src/float.cr

```diff
@@ -155,6 +155,22 @@ struct Float32
     LibM.round_f32(self)
```

Comment:
This round method should go away and use Number overload to define the default, right?

Reply:
Yes, gould catch. I had actually just fixed that before publishing the PR but it seems I didn't push that change.

Reply:
Ooops, sorry for the force push.